src/Common/src/Interop/Unix/System.Native/Interop.MAdvise.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+
+using size_t = System.UIntPtr;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, SetLastError = true)]
+        internal static extern int MAdvise(IntPtr addr, size_t length, MemoryAdvice advice);
```

Comment:
If we've decided to use uintptr_t / UIntPtr for size_t, can we just do so in our signatures on both the managed and native sides and not do the "using size_t = System.UIntPtr;" thing?